### DISCUSSION TITLE
State of the Facebook Group

### DISCUSSION THREAD

**Discussion by e-hart** [TheLazyGeekGuy/DIY-Waves-Soundgrid-Server#7] [General]
The Facebook group has been a great resource over time, but Kalfelz Wusik William has taken to using it as his own promotional pipeline for his own music, abusing @everyone tags multiple times a week. He instantly bans users who so much as question it.

Today I posted a simple comment suggesting that the practice would be damaging to the community over time. I very quickly (in a matter of about a minute or two) got about 6 likes on the comment and just moments later, I was banned.

I am now left without a useful online community for this topic. The Facebook is the main discussion board and best place to get quick answers from people, and to have an admin who reacts so heavily to *perceived* personal slights is unsustainable.

I'd encourage @TheLazyGeekGuy to ask Kalfelz Wusik William about this, as it very much does seem that his admin powers are being abused for his own whims.

Moving forward, I wonder if @TheLazyGeekGuy would be willing to open and administrate a Discord server where community interaction can occur without inappropriate admin interference.

**Comment by TheLazyGeekGuy** [comment]
First of all, running a group alone is a lot of work and KW has done a lot of incredible work on his spare time.

Most of facebook group applies and ties to a general rules published and agreed by anyone joining the group; and for the good of everyone - most of the time ban any kind of personal promotion that lead to the opposite to an good communication.

But facebook group owner is the only one to have the liberty to decide what is inside; anyone can fork it to a new one - but it is not a good approach.

Waves DIY SoundGrid Server admin have been warned of thoses arguments :
- there is too much content out of context
- at some point the group status should clarified, rules or guideline.

I'm pretty confident @MildlyAggressiveFolk that all is going back to normal in the next few days.

Wait and see !

**Comment by WilliamKWusik** [comment]
So sad, very sad to see people complaining about this, after all the work I put EVERY DAY! People are selfish, musicians don't care about each other, it's just sad. I guess it's time to close the group. :-\

**Comment by e-hart** [comment]
Do you mind sharing the contents of the post that you consider to be
grounds for my banning? I no longer have access to that.

**Comment by e-hart** [comment]
In case you claim not to have access to it: I simply suggested in a
courteous manner that it might be a bit inappropriate to abuse @everyone
notification blasts for purposes unrelated to the group. This has nothing
to do with appreciating your work (which I do) but rather your conduct.

It isn’t a music group, it is a group of thousands of professionals and
hobbyists looking for a place to collate information. @everyone tags cause
push notifications to over 4000 people. It’s simply not appropriate to do
that on a regular basis for the purposes of your own self-promotion.

You banned me simply for starting a conversation about something that MANY
others in the group agreed with. Over 6 likes on my comment in just a
minute or two, then you turned off comments on the post because of it.

I now no longer have access to a valuable online resource that is one of
the only places online to troubleshoot these systems with other
individuals. All because you got your feelings hurt when I talked to you
like an adult.

That childish behavior will rot the community from the inside whether you
end the group or not, but if you wanted to be reasonable about it, you
could allow someone else to admin the group and simply step away instead of
vandalizing a tool that many professionals worldwide use on your way out
the door.

**Comment by e-hart** [comment]
Still banned. @WilliamKWusik, I'd like to invite you once again to publicly share the contents of my comment which was deemed ban-worthy.

Your behavior here is childish. Small men and their tiny kingdoms, I guess.
